tests/settings.py

```diff
@@ -21,14 +21,16 @@
     # local sqlite database file
     DATABASES['default'] = {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': INSTALL_DIR + 'db.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
What is the reasoning behind this change, i.e. moving test database out of the install directory and into the tests directory?